src/styles.css

```diff
@@ -264,6 +265,26 @@ h3 {
   height: 97%;
 }
 
+.individual-recipe-image {
+  position: absolute;
+  bottom:0;
+  right:0;
+  flex-shrink: 0;
+  width: 200px;
+  height: 200px;
+  overflow: hidden;
+  border-radius: 50%;
+  z-index: 1;
+  margin-right: -40px;
+}
+
+.individual-recipe-image img {
+  display: inline;
+  margin: 0 auto;
```

Comment:
Could you describe what margin: 0 auto does?

Reply:
Likely is doing nothing XD.  I thought I was needing to overwrite a margin property, but looks like I don't anymore. Removing.